docs/examples/.env.truenas.example

```
@@ -0,0 +1,64 @@
+
+# ============================================
+# EMAIL (uses Mailpit by default for staging)
+# ============================================
+SMTP_HOST=mailpit
+SMTP_PORT=1025
+SMTP_USER=
+SMTP_PASSWORD=
+SMTP_TLS=false
+SMTP_SSL=false
+EMAIL_FROM_ADDRESS=noreply@falloutshelter.com
+EMAIL_FROM_NAME=Fallout Shelter
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_FROM_NAME key should go before the SMTP_HOST key

(UnorderedKey)